package.json

```diff
@@ -4,32 +4,48 @@
   "description": "Proxy that handles interactions with pagoPA",
   "main": "index.ts",
   "repository": "https://github.com/teamdigitale/italia-pagopa-proxy",
-  "author": "Federico Feroldi <federico@teamdigitale.governo.it>",
+  "author": "Sergio Manetta <sergio.manetta@it.ibm.com>",
   "license": "MIT",
   "scripts": {
-    "build": "npm run lint && tsc",
-    "test": "jest",
-    "lint": "tslint --project ."
```

Comment:
it's useful to have a separated `lint` task

Reply:
Added into next push